src/components/snapshot/overview/SnapshotOverviewPanel.test.tsx

```diff
@@ -69,26 +107,29 @@ const initialState = {
   },
 };
 
+function mockSnapshotOverviewPanel(snapshot: any): any {
```

Comment:
Just making sure I understand - you're creating this function instead of using the "BeforeEach" method so that you can pass in a different snapshot for a given test, right?

Reply:
Exactly!  But I am open to a different way to achieve this.